_data/tools.yml

```
@@ -100,3 +100,25 @@ development:
       - name: Selenium
         documentation: https://www.seleniumhq.org/
     
+- focus: ESDC Standard Security Tools by Security Testing Type
+    scope: SAST (Legacy & Cloud)
+    tools:
+      - name: Microfocus Fortify Static Code Analyzer
+        documentation: https://www.microfocus.com/en-us/products/static-code-analysis-sast/overview
+    scope: DAST (Legacy & Cloud)
```

We'll also need to add the focus in each time... (and the space is nice for separation 😃)
```suggestion

  - focus: ESDC Standard Security Tools
    scope: DAST (Legacy & Cloud)
```